typewriter/fixes/fixer_utils.py

```diff
@@ -53,25 +122,42 @@ def type_by_import_stmt(package, name, node):
     return None
 
 
-def create_import(package, name, node):
-    # type: (str, str, Node) -> None
-    """
-    Create import statement of the form `from <package> import <name>`
+def is_import_stmt(node):
+    return (node.type == syms.simple_stmt and node.children and
+            is_import(node.children[0]))
 
-    Parameters
-    -------------
-    package : str
-    name : str
-        Name of type being imported
-    node : Node
-    """
 
-    def is_import_stmt(node):
-        return (node.type == syms.simple_stmt and node.children and
-                is_import(node.children[0]))
+def _generate_import_node(package, name, prefix=""):
+    if package is None:
+        import_ = Node(syms.import_name, [
+            Leaf(token.NAME, "import", prefix=prefix),
+            Leaf(token.NAME, name, prefix=" ")
+        ])
+    else:
+        import_ = Node(syms.import_from, [
```

Comment:
For some reason the `FromImport` function does not have a `prefix` kwarg. This is an issue for our use case because we need granular control over the prefix of the import statement.